internal_ws/ykcompile/src/lib.rs

```diff
@@ -264,7 +264,9 @@ fn local_to_reg_name(loc: &Location) -> &'static str {
 // interpreter for blackholing.
 
 /// Given a vector of `FrameInfo`s `vptr`, instantiates and initialises a SIR interpreter and returns its
-/// pointer. Consumes `vptr`.
+/// pointer. Consumes `vptr`. XXX note this requires a move of `vptr`, so we should document
+/// something like "note that the calling function must fully transfer ownership of vptr
+/// to this function".
```

Comment:
It says it consumes `vptr`. That's the idiom we've used so far in docs.

Reply:
Yes, this comment is wrong -- though there are other functions which don't document "consume". I must have done this one on auto-pilot though!